src/lib/cli/rules/templates/external-rule/files/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,2 @@
+This project follows the [JS Foundation Code of
```

Comment:
We don't need this file. We used to have it because it was a repo in `sonarwhal` but it's not needed now.